features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Onboarding.jsx

```diff
@@ -101,25 +101,43 @@ function Onboarding() {
                     description={(
                         <FormattedMessage
                             id='ServiceCatalog.Listing.Onboarding.learn.heading.text'
-                            defaultMessage='Create and Deploy your first Integration Service'
+                            defaultMessage={'Create and Deploy your first Integration Service with WSO2 Micro'
+                            + ' Integrator or Streaming Integrator'}
                         />
                     )}
                 >
+                    &nbsp;&nbsp;
                     <Button
                         className={classes.actionStyle}
-                        size='large'
+                        size='small'
                         variant='outlined'
                         color='primary'
                         href='https://apim.docs.wso2.com/en/4.0.0/design/create-api/create-an-api-using-a-service/'
+                        endIcon={<LaunchIcon style={{ fontSize: 15 }} />}
+                    >
+                        <FormattedMessage
+                            id='ServiceCatalog.Listing.Onboarding.learn.link.micro.integrator'
+                            defaultMessage='Micro Integrator'
+                        />
+                    </Button>
+                    &nbsp;&nbsp;
```

Comment:
Shall we avoid using &nbsp; here and use {' '} instead? &nbsp; does not seem to be working in some browsers.

Reply:
Fixed with 34700c2a7356a5d035adb21e29659a476a6a07b7

Reply:
Fixed with 2e937d4af09e1e8547f26dc5bffb8962e6156a3e